### ISSUE TITLE
Set up dependency updates

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/dataloading-management#24]
- [x] Add `.autoupdate/postupdate` (use [Argo's](https://github.com/sul-dlss/argo/blob/main/.autoupdate/postupdate) as an example): https://github.com/sul-dlss/dataloading-management/pull/34
- [x] Add to access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/219
- [x] ~~Add to sdr-deploy~~ (This will become a new ticket blocked by having servers provisioned.)
